no-computer.md

```diff
@@ -11,41 +11,105 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Gymnastics, diving, and figure skating are Olympic sports with seemingly 
+ambiguous scoring systems. Part of the ambiguity comes from the subjective 
+nature of artistic sports, but the part comes from spectators who don't know 
+how scoring works and therefore make a hissy fit when their favorite people 
+lose.
+
+While I can't stop hissy fits, I will show how a DSL can help rank moves and 
+how that translates into points. For now, I will focus on gymnastics since it 
+is the most popular "subjective" Olympic sport.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+This isn't just a rulebook. A language can help describe the moves (and its 
+corresponding point values). Moreover, shaky transitions can lower the score 
+and spiral into poorly executed moves. Essentially, I want to clarify how 
+scores are calculated and how each move contributes to he overall score. This 
+idea is very domain-specific, as it only pertains to gymnastics or other 
+artistic sports.
+
```

Comment:
Interesting! Never really thought of this as something possible to be improved since it seems so "human" and inherently subjective. But I think you're right, if we can measure and rank certain moves and the shakiness, etc. then it certainly could be improved.